tools/github/github-utils.ts

```diff
@@ -53,16 +52,23 @@ export async function getCommitAndLabels(
 
   const prByLabels = {};
   for (const commit of commits) {
-    const prs = await octokit.rest.repos.listPullRequestsAssociatedWithCommit({
-      owner,
-      repo,
-      commit_sha: commit.sha,
-    });
-    for (const pr of prs.data) {
-      if (pr.labels && pr.labels.length > 0) {
-        for (const label of pr.labels) {
+    const commitMessageFirstLine = commit.commit.message.split("\n")[0];
+    const foundPrsNumbers = commitMessageFirstLine.match(/\(#([0-9]+)\)/);
+    if (
+      !commitMessageFirstLine.startsWith("Revert ") &&
```

Comment:
I think we still want to display the revert commits, but we don't want to include the commit that is reverted.

Reply:
If a commit is reverted in the same version as the original commit, I think it is better to hide both.